build.gradle

```diff
@@ -1,52 +1,22 @@
+import qsl.internal.Versions
+
 plugins {
 	id("maven-publish")
-	id("org.quiltmc.gradle-conventions.qsl.common")
+	id("qsl.common")
 }
 
 group = "org.quiltmc.qsl-prototype"
-version = project.qsl_version
-
-def coreDependencies(Project project, List<String> depNames) {
-	_moduleDependencies(project, depNames.iterator().collect { "core:$it" })
-}
+version = Versions.QSL_VERSION
 
-def libraryDependencies(Project project, List<String> depNames) {
-	_moduleDependencies(project, depNames.iterator().collect { "$project.qsl_library:$it" })
-}
-
-private def _moduleDependencies(Project project, List<String> depNames) {
-	def deps = depNames.iterator().collect {
-		project.dependencies.project(path: ":$it", configuration: "dev")
-	}
-
-	project.dependencies {
-		deps.each {
-			implementation it
-		}
-	}
-
-	project.publishing {
-		publications {
-			mavenJava(MavenPublication) {
-				pom.withXml {
-					def depsNode = asNode().appendNode("dependencies")
-					deps.each {
-						def depNode = depsNode.appendNode("dependency")
-						depNode.appendNode("groupId", it.group)
-						depNode.appendNode("artifactId", it.name)
-						depNode.appendNode("version", it.version)
-						depNode.appendNode("scope", "compile")
-					}
-				}
-			}
-		}
+sourceSets {
+	testmod {
+		compileClasspath += main.compileClasspath
+		runtimeClasspath += main.runtimeClasspath
 	}
 }
 
-jar {
```

Comment:
you have a `jar { enabled = false }` for the libraries but not here, which is it?

Reply:
Probably need to disable it here. Might have over done it

Reply:
Done